swap_meet/vendor.py

```diff
@@ -1,2 +1,92 @@
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+
+        self.inventory.remove(item)
+        return item
+
+    def get_by_category(self, category):
+        items_in_category = []
+        for thing in self.inventory:
```

Comment:
I would choose a word a little for descriptive than a thing 😂. The word item could work fine here.